### ISSUE TITLE
karma web test suite doesn't work with ts_project

### ISSUE THREAD

**Issue by bduffany** [bazelbuild/rules_nodejs#2518]
When using `karma_web_test_suite` from `@npm//@bazel/concatjs:index.bzl` along with `ts_project`, I'm getting the following error:

```
HeadlessChrome 76.0.3809 (Linux 0.0.0): Executed 0 of 0 ERROR (0.001 secs / 0 secs)
```

The full log when running with `--test_output=streamed` is in [this paste](https://pastebin.com/yewwkkn3)

Discovered this while trying to update the web testing example in this repo to use `ts_project`. See the PR here: https://github.com/bazelbuild/rules_nodejs/pull/2516

**Env**: `rules_nodejs` at HEAD, `bazel 4.0.0`, `Ubuntu 20.04`

**Comment by alexeagle** [comment]
yeah the `@bazel/concatjs` package is documented to require that all inputs be named UMD modules. That's what `ts_library` produces, so following the google3 model those are frequently used together.

It's possible to configure ts_project to produce named UMD outputs, like we do in the concatjs package itself
https://github.com/bazelbuild/rules_nodejs/blob/3.2.2/packages/concatjs/BUILD.bazel#L28
(and as required by TypeScript, you have to name the module yourself
https://github.com/bazelbuild/rules_nodejs/blob/3.2.2/packages/concatjs/index.ts#L1 )

But for most users we don't recommend these google-specific idioms, and so maybe you want to use a more modern/maintained testing library
